ddtrace/contrib/pylons/middleware.py

```diff
@@ -70,13 +77,10 @@ def _start_response(status, *args, **kwargs):
                 # e.code can either be a string or an int
                 code = getattr(e, "code", 500)
                 try:
-                    code = int(code)
-                    if not 100 <= code < 600:
-                        code = 500
-                except Exception:
+                    int(code)
+                except (TypeError, ValueError):
```

Comment:
This pattern of setting a default status code (even if one is not included in the response) is not conventional within the library but I retained it in the event that users were dependent on it with alerts/monitors